manuscript/template.tex

```diff
@@ -483,14 +485,16 @@ \subsection{Data Structures}
 \unskip
 
 \subsection{Document Generation}
-As noted multiple times throughout this article, there have been a number of figures and tables generated from the YAML files placed within the SeaLion mission architecture GitHub repository.  Many of the figures are UML diagrams that are auto-generated artifacts rendered from the M30ML modeling language and formatted using the Liquid template language.  This is the link for the docs-as-code approach.  YAML code files are used to generate documents for information sharing between group members.  This means that any changes made to the SeaLion mission architecture model can immediately be used to generate new documents.  Whether it be diagrams, tables, or text, continuous updating is ensured that any changes affecting dependencies within the mission architecture are kept in sync.  A conference proceeding manuscript presented in AIAA SciTech 2023 was created purely by a docs-as-code format \cite{scitech_proceeding}.  The team used a LaTeX template to automatically format the manuscript to the conference guidelines and subsequently inject items such as the generated diagrams, tables, and references directly into the manuscript.
+As noted multiple times throughout this article, there have been a number of figures and tables generated from the YAML files placed within the SeaLion mission architecture GitHub repository.  Many of the figures are UML diagrams that are auto-generated artifacts rendered from the M30ML modeling language and formatted using the Liquid template language.  This is the link for the docs-as-code approach.  YAML code files are used to generate documents for information sharing between group members.  This means that any changes made to the SeaLion mission architecture model can immediately be used to generate new documents.  Whether it be diagrams, tables, or text, continuous updating is ensured that any changes affecting dependencies within the mission architecture are kept in sync.  YAML files are automatically processed using templates langauge (e.g., Jinja2) via a build shell script.  This also means that content of the model and formatting of documents are decoupled.  The model is formatting-agnostic for documentation purposes as this is handled by template language.  A conference proceeding manuscript presented in AIAA SciTech 2023 was created purely by a docs-as-code format \cite{scitech_proceeding}.  The team used a LaTeX template to automatically format the manuscript to the conference guidelines and subsequently inject items such as the generated diagrams, tables, and references directly into the manuscript.
```

Comment:
can remove `This is the link for the docs-as-code approach.`

Reply:
`by template language` s/b `by the template language`

Reply:
Resolved dda6156f73e0ae1611534d1311a34f2b1af3c6fe